R/score.R

```diff
@@ -106,7 +114,14 @@ score <- function(data,
                   ...) {
 
   # preparations ---------------------------------------------------------------
-  check_data <- check_forecasts(data)
+  if (is(data, "scoringutils_check") &&
+    !any(is.null(data$cleaned_data), is.null(data$prediction_type),
```

Comment:
could implect a check_scoringutils_check internal for this as may be useful elsewhere.

Reply:
I implemented a `is.scoringutils_check()` function. Maybe your suggested name is better - the function doesn't only check for class inheritance, but also checks whether the required list items are available. I'm not sure that additional check is necessary, but maybe users do crazy and unexpected things...

Reply:
ah I take that back its an s3 method. That is nice.

Reply:
or wait did you make it an s3 method??

Reply:
yep, a while ago :) It's the only S3 method scoringutils has :-D

Reply:
(generally open to more S3 as long as it doesn't make it harder to use for the user :))